test/rest/2023_07/discount_code_test.rb

```diff
@@ -324,4 +324,26 @@ def test_9()
     end
   end
 
+  sig do
```

Comment:
I only put this one tests in the latest API version, I doubt this is where this test belongs as it's supposed to be auto-generated. Seems to me that the auto-generation of these tests are missing some error cases.